queue/queue.go

```diff
@@ -0,0 +1,32 @@
+package party
+
+import (
+)
+
+type Queue struct {
+	 songList []string
+}
+
+func NewQueue() *Queue {
+	q := Queue{
+		songList:make([]string, 10),

```

Comment:
2nd on the making a class / type of some kind, maybe check out [user](https://github.com/me-next/menext-backend/blob/develop/party/user.go) where there is a uuid type for users and a "class" for user